File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1044_basico.js
import { question } from "readline-sync";

function main() {
    const valores = get_number();

    let Nvalores = valores.split(` `);
    let A = parseInt(Nvalores[0]);
    let B = parseInt(Nvalores[1]);

    const resposta = Eh_Multiplo(A, B);

    print(`${resposta}`);
}

function get_number() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

function Eh_Multiplo(num1, num2) {
    let mensagem;
    
    if (num2 % num1 === 0 || num1 % num2 === 0) {
        mensagem = `Sao Multiplos`
    } else {
        mensagem = `Nao sao Multiplos`
    }
    return mensagem
}

main();

File: Exercicios_ED_I/Bee/1044_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    int a, b;
    cin >> a >> b;

    if(b % a == 0 || a % b == 0) {
        cout << "Sao Multiplos" << endl;
    } else {
        cout << "Nao sao Multiplos" << endl;
    }
}